language/fr/alliance.lang

```diff
@@ -1,5 +1,575 @@
+
+$_Lang['AFP_Name']                  = 'Nom';
+$_Lang['AFP_Tag']                   = 'Tag';
+$_Lang['AFP_Members']               = 'Membres';
+$_Lang['AFP_YourRank']              = 'Ton rang';
+$_Lang['AFP_WebPage']               = 'Site internet';
+$_Lang['AFP_ExternalSec']           = 'Description publique';
+$_Lang['AFP_InnerSec']              = 'Description privée (réservée aux membres)';
+
+$_Lang['AFP_NoAllyDesc']            = &$_Lang['AInfo_NoAllyDesc'];
+$_Lang['AFP_NoAllyInner']           = 'Aucune description privée.';
+
+$_Lang['AFP_MemberOptions']         = 'Options disponibles';
+$_Lang['AFP_MembersList']           = 'Membres';
+$_Lang['AFP_InvitesList']           = 'Invitations';
+$_Lang['AFP_InviteUser']            = 'Inviter un joueur';
+$_Lang['AFP_PactsList']             = 'Pactes';
+$_Lang['AFP_SendMail']              = 'Envoyer un message aux membres';
+$_Lang['AFP_AllyChat']              = 'Discussion en ligne';
+$_Lang['AFP_AllyAdmin']             = 'Gestion';
+$_Lang['AFP_ReqList']               = 'Candidatures';
+
+$_Lang['AFP_LeaveAlly']             = 'Quitter l\'alliance';
+$_Lang['AFP_LeaveButton']           = 'Clique ici pour quitter l\'alliance';
+$_Lang['AFP_SureWantLeave']         = 'Es-tu sûr de vouloir quitter l\'alliance ?';
+
+$_Lang['AFP_CollExp']               = 'Déplier / Replier';
+
+// ---------------------------------
+
+// Ally Admin - Front Page
+
+$_Lang['ADM_Title']              = &$_Lang['AFP_AllyAdmin'];
+
+$_Lang['ADM_FrontPage']          = 'Paramètres généraux';
+$_Lang['ADM_AllyInfo']           = 'Modifier le profil de l\'alliance';
+$_Lang['ADM_MembersList']        = 'Gestion des membres';
+$_Lang['ADM_ChangeRanks']        = 'Gestion des rangs';
+$_Lang['ADM_ReqList']            = 'Gestion des candidatures';
+
+$_Lang['ADM_Texts']              = 'Gestion des textes';
+$_Lang['ADM_ExternalText']       = 'Description publique';
+$_Lang['ADM_InternalText']       = 'Description privée';
+$_Lang['ADM_RequestText']        = 'Exemple de candidature';
+
+$_Lang['ADM_Characters']         = &$_Lang['Ally_WR_Chars'];
+$_Lang['ADM_Clear']              = &$_Lang['Ally_WR_Clear'];
+$_Lang['ADM_SaveAll']            = 'Tout enregistrer';
+$_Lang['ADM_SaveOne']            = 'Enregistrer ';
+$_Lang['ADM_SaveMark1']          = $_Lang['ADM_SaveOne'].'la description publique';
+$_Lang['ADM_SaveMark2']          = $_Lang['ADM_SaveOne'].'la description publique';
+$_Lang['ADM_SaveMark3']          = $_Lang['ADM_SaveOne'].'l\'exemple de candidature';
+
+$_Lang['ADM_ReqSettings']        = 'Paramètres des candidatures';
+$_Lang['ADM_AllyOpen4Req']       = 'Accepter de nouvelles candidatures ?';
+$_Lang['ADM_NewComerRank']       = 'Rang par défaut des nouveaux membres';
+$_Lang['ADM_AllyOpen4ReqInfo']   = 'Quand cette option est définie à "Non", personne ne pourra soumettre de nouvelles candidatures.<br/><b>(Toutes les candidatures déjà reçues seront automatiquement refusées !)</b>';
```

Comment:
According to an online translator, this mentions that already received applications will be automatically rejected. Both English and Polish translations mention that already received applications **won't** be automatically rejected (you have to reject them manually).

Is there a translation mistake, or is the translator wrong?

Reply:
My mistake. Fixed.